Dockerfile

```diff
@@ -11,6 +11,7 @@ ENV PATH /app/node_modules/.bin:$PATH
 COPY package.json package-lock.json /app/
 
 # install dependencies
+RUN apk add --no-cache git
```

Comment:
Why do we need git inside the container?

Reply:
Because sometimes `npm` needs to fetch dependencies directly from Git, which causes `npm install` to fail in CI.